src/odemis/driver/pigcs.py

```diff
@@ -4224,9 +4299,23 @@ def _processCommand(self, com):
                 axis = int(args[1])
                 if axis != 1:
                     raise SimulatedError(15)
+
+                # simulate moving to reference position
+                ref_pos = self._parameters[0x16] * 1e3  # value at reference
+                # TODO I have not idea why it is necessary to multiply by *1e3,
+                #  but saw it also happen in the MOV commands that values were in mm instead of m.
```

Comment:
Yes, exactly, the parameters are stored in mm, but internally the computations are done in m.